tests/models/gptsan_japanese/test_modeling_gptsan_japanese.py

```diff
@@ -151,6 +151,12 @@ def test_model(self):
         config_and_inputs = self.model_tester.prepare_config_and_inputs()
         self.model_tester.create_and_check_model(*config_and_inputs)
 
+    @unittest.skip(
+        reason="skip for now as the computed `max_memory` by `model_split_percents` in the test method will be changed inside `from_pretrained`"
+    )
```

Comment:
Skip for now as discussed offline.

Changing `model_split_percents` won't work, as `max_memory` will be changed in `from_pretrained` and that modified value makes all model modules being on GPU 1. 

```python
            if device_map != "sequential" and get_balanced_memory is not None:
                max_memory = get_balanced_memory(
                    model,
                    max_memory=max_memory,
                    no_split_module_classes=no_split_modules,
                    dtype=torch_dtype,
                    low_zero=(device_map == "balanced_low_0"),
                )
```